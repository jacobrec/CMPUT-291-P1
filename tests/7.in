1
jacob
password
1
1
o
p
l
m
;
0
n
1
16
y
YYYY-MM-DD
2016-00-00
2016-01-00
2016-13-31
2016-12-32

Going too fast
-10
1
0
16
y
2016-08-08
Being a jerk
1000
3
2
