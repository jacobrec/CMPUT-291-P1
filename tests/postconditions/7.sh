echo "SELECT regno, fine, violation FROM tickets WHERE vdate=DATE('$(date "+%Y-%m-%d")')" | sqlite3 src/sql/test.db
echo "SELECT regno, fine, violation, vdate FROM tickets WHERE vdate=DATE('2016-08-08')" | sqlite3 src/sql/test.db
