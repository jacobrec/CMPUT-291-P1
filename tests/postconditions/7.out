regno       fine        violation
----------  ----------  --------------
16          -10         Going too fast
regno       fine        violation     vdate
----------  ----------  ------------  ----------
16          1000        Being a jerk  2016-08-08
